config.json

```diff
@@ -122,6 +122,14 @@
         "concepts": ["while-loops", "return"],
         "prerequisites": ["conditionals"],
         "status": "beta"
+      },
+      {
+        "slug": "library-of-luton",
+        "name": "Library of Luton",
+        "uuid": "d98302fa-fd76-469d-83dd-235c71363b43",
+        "concepts": ["char"],
+        "prerequisites": ["while-loops"],
```

Comment:
I think the user also needs to know about `strings`?

Reply:
Yeah, I use my mermaid map, so I just set the prerequisites to the concept right before, I don't know if that is the right way though.

Reply:
If `while-loops` has `strings` in its prerequisites (or indirectly), that's fine!